microsoft-edge/webview2/reference/HoloLens2/webview2-unity-plugin.md

```diff
@@ -0,0 +1,316 @@
+---
+title: WebView plugin for Unity API Reference (Preview)
+description: API Reference for Microsoft Mixed Reality WebView plugin for Unity.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/08/2023
+---
+# Mixed Reality WebView Plugin for Unity API Reference (Preview)
+The **Microsoft Mixed Reality WebView plugin for Unity** enables the integration of WebView2 functionality into your HoloLens 2 app.  This WebView plugin for Unity simplifies the integration of WebView2 functionality into your HoloLens 2 app by wrapping the WebView2 control, automatically handling rendering, and automatically directing input to the WebView2 control.
+
+This plugin also manages interop between Unity and WebView2, enabling communication between JavaScript and Unity via messages and events.
+
+This plugin exposes a subset of the functionality available via [CoreWebView2](/dotnet/api/microsoft.web.webview2.core.corewebview2).
+
+WebView2 on HoloLens 2 and the WebView plugin for Unity are both in Preview and are subject to change before general availability.  The WebView2 Preview is available in the Insider Preview for Microsoft HoloLens.  To access this preview, you must be enrolled in the Windows Insider Program; see [Start receiving Insider builds](/hololens/hololens-insider#start-receiving-insider-builds) in _Insider preview for Microsoft HoloLens_.
+
+WebView2 and the WebView plugin are only supported on HoloLens 2 devices running the Windows 11 update. For more information, see [Update HoloLens 2](/hololens/hololens-update-hololens).
+
+For information on getting started with the plugin, see [Get started with WebView2 in HoloLens 2 Unity apps](/microsoft-edge/webview2/get-started/hololens2.md).
+
+### IWebView Interface

```

Comment:
```suggestion
### IWebView interface
```
Not sure what style convention to use.
Lowercase b/c:
* corp style is sentence case for titles/headings.
* C++ look & feel (if relevant) favors lowercase in API Ref docs.